lib/tasks/trends.rake

```diff
@@ -1,4 +1,16 @@
 namespace :trends do
+  desc 'Clean up Trends data'
+  task :clobber => :environment do
+    class FakeTrend < ApplicationRecord
+      self.table_name = 'trends'
+    end
+    class FakeTrendCounter < ApplicationRecord
+      self.table_name = 'trend_counters'
+    end
+    FakeTrendCounter.delete_all
+    FakeTrend.delete_all
+  end
```

Comment:
We need to also cleanup the db tables, permissions and settings when running this task. Should we put it in a different namespace where we can put additional cleanup tasks in the future?

Reply:
My :heart: hurts handling schema anywhere else then migrations. Do we really want to drop the tables from the task? Given there should be a migration that will drop all the data for everyone.
Setting and perms I agree :100:  :+1:

Reply:
I'm not sure about dropping it for everyone in migration, what if i upgraded and didn't mean to remove the data but only realized that i need to install the plugin too late?

Reply:
well that's a bummer and tough life but that is what it is... You've had two versions to realize that.
I mean assuming we are going to add the migration with droping the deprecated endpoints.

Reply:
I think it makes sense to drop here as well - when someone runs this task, they are saying "we don't want this stuff, get rid of it", which to me means we should also drop the tables. We can still add a migration as well to make sure we remove them when we drop the api endpoints for those who were lazy and didn't care about running the task.